src/test/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImportIntegrationTest.java

```diff
@@ -455,14 +551,52 @@ public void testPreserveContigOrderingInHeader() throws IOException {
 
     }
 
+    private static String getQueryJsonForGenomicsDB(String vidMappingFile, String callsetMappingFile, String tiledbWorkspace,
+                                                     String referenceGenome, boolean produceGTField) throws IOException {
+        //Produce temporary JSON query config file
+        String indentString = "    ";
+        String queryJSON = "{\n";
+        queryJSON += indentString + "\"scan_full\": true,\n";
+        queryJSON += indentString + "\"workspace\": \""+tiledbWorkspace+"\",\n";
+        queryJSON += indentString + "\"array\": \""+GenomicsDBConstants.DEFAULT_ARRAY_NAME+"\",\n";
+        queryJSON += indentString + "\"vid_mapping_file\": \""+vidMappingFile+"\",\n";
+        queryJSON += indentString + "\"callset_mapping_file\": \""+callsetMappingFile+"\",\n";
+        queryJSON += indentString + "\"produce_GT_field\": true,\n";
```

Comment:
This piqued my interest for a totally unrelated reason -- is this how you get the GDB to echo back the GTs that went in instead of converting to ./.?  Or is it an option to produce sites-only output?  Either one would be useful.

Reply:
Its what you need to get the GTs back. Unfortunately for the time being it has to be done with the query json file, though we have asked for a easier api solution so that might eventually show up.